### PR TITLE
Hygiene: Replace eslint-config-node-services with eslint-config-wikimedia

### PR DESCRIPTION
**Pull request by mdholloway**
Drops eslint-config-node-services in favor of the new 'server' preset in eslint-config-wikimedia, and applies most rules.

Upstreams https://gerrit.wikimedia.org/r/#/q/topic:eslint-config-wikimedia.

### CODE REVIEW

**Review by jdforrester — APPROVED** [review]

**Comment by jdforrester** [comment]
(Should get a +1 from @Pchelolo before merging.)

**Comment by Pchelolo** [comment]
replacing spaces with tabs.... heh.. that would b a big one. I don't know if we should do that tbh but I'm certain we need more discussions here. @d00rman must have an opinion as well. 

My personal opinion is that in this particular case losing git blame history might be a deal breaker.

**Comment by Nikerabbit** [comment]
There is `git blame -w` that ignores whitespace changes.

**Comment by Pchelolo** [comment]
That is true, however if we merge this we get ourselves into using git blame -w forever. Plus, imagine how painful would it be to merge this change into every service that was forked from the template. Tabs vs spaces is a holy war I do not want to begin, but I am against changing one to another for no specific reason. We are making a lot of overrides in the config - why not overwrite the tabs rule?

**Comment by mdholloway** [comment]
OK, I've dropped the controversial commit from this PR and opened https://phabricator.wikimedia.org/T211138 for discussion.

**Comment by mdholloway** [comment]
On a side note, Yuri pointed out in a [Tilerator PR](https://github.com/kartotherian/tilerator/pull/32#pullrequestreview-134518635) that we'd really be better off putting code that's likely to be updated periodically into node modules, and using the template as strictly a starting point.  That seems like a good point, and doing this would lower the stakes for the code style of the service template since service maintainers wouldn't be required to adhere to the same code style just to merge in future changes.

**Comment by mdholloway** [comment]
Looks like this has been superseded. https://github.com/wikimedia/service-template-node/commit/07acb8ac15cac27ae5a8ecbb9141ba20b6883140

**Pull request closed without merging**
